AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3845,63 +3847,63 @@ function Debug-AzStorageAccountEntraKerbAuth {
         }
         #
         # Port 445 check 
-        #
-        
+        #        
         if (!$filterIsPresent -or $Filter -match "CheckPort445Connectivity")
         {
+            Write-Host "Checking Port 445 Connectivity"
             try {
+                Write-Host "`tTesting ... " -NoNewline
                 $checksExecuted += 1;
-                Write-Verbose "CheckPort445Connectivity - START"
-
-                Test-Port445Connectivity -StorageAccountName $StorageAccountName `
-                    -ResourceGroupName $ResourceGroupName -ErrorAction Stop
-
+                Test-Port445Connectivity -StorageAccountName $StorageAccountName -ResourceGroupName $ResourceGroupName -ErrorAction Stop
                 $checks["CheckPort445Connectivity"].Result = "Passed"
-                Write-Verbose "CheckPort445Connectivity - SUCCESS"
+                Write-TestingPassedPSStyle
             } catch {
+                Write-TestingFailedPSStyle
                 $checks["CheckPort445Connectivity"].Result = "Failed"
                 $checks["CheckPort445Connectivity"].Issue = $_
-                Write-Error "CheckPort445Connectivity - FAILED"
-                Write-Error $_
+                Write-FailedPSStyle $_

```

Comment:
It seems like almost all the calls to `Write-TestingFailedPSStyle` are followed by a call to `Write-FailedPSStyle`. We could maybe consider taking the message directly in `Write-TestingFailedPSStyle`, so that we don' thave to follow up with `Write-FailedPSStyle`